packages/common/src/constants.ts

```diff
@@ -18,6 +18,7 @@ export const REGEX_SRC_FILE = /\.[cm]?[jt]sx?$/
 export const REGEX_SETUP_SFC = /\.setup\.[cm]?[jt]sx?$/
 export const REGEX_VUE_SFC = /\.vue$/
 export const REGEX_VUE_SUB = /\.vue\?vue&type=script/
+export const REGEX_VUE_DIRECTIVE = /\s(v-if|v-for)=/
```

Comment:
It's unnecessary to extract to the common package. It will only be used in `jsx-directive` package.